README.md

```diff
@@ -1,9 +1,26 @@
 # Pern-test
```

Comment:
I'm missing how to run the migrations in the document, could you add it (Both in local and docker deployments)

Reply:
Implemented in the last commits